patchkit-launcher-qt-src/src/launcherworker.cpp

```diff
@@ -219,7 +219,15 @@ void LauncherWorker::runWithData(Data& t_data)
     }
 
     lockFile.cede();
-    startPatcher(t_data);
+    try
```

Comment:
Remember to put note about this in change log since it's not related to pull request changes.